tests/plugins/autocomplete/autocomplete.js

```diff
@@ -40,18 +54,22 @@
 			editor.editable().fire( 'keyup', new CKEDITOR.dom.event( {} ) );
 
 			assertViewOpened( ac, true );
+			assert.isTrue( ac.view.getItemById( 1 ).hasClass( 'cke_autocomplete_selected' ) );
```

Comment:
Again, you're asserting here same thing as in "test autocomplete starts with the first item selected" TC. This is not needed.